jib-core/src/integration-test/java/com/google/cloud/tools/jib/api/JibIntegrationTest.java

```diff
@@ -220,19 +222,44 @@ public void testBasic_tarBaseImage_jibImageToDockerDaemon()
   }
 
   @Test
-  public void testScratch()
+  public void testScratch_singlePlatform()
       throws IOException, InterruptedException, ExecutionException, RegistryException,
           CacheDirectoryCreationException {
     ImageReference targetImageReference =
         ImageReference.of("localhost:5000", "jib-core", "basic-scratch");
-    Jib.fromScratch().containerize(getLocalRegistryContainerizer(targetImageReference));
+    Jib.fromScratch()
+        .setPlatforms(ImmutableSet.of(new Platform("arm64", "linux")))
```

Comment:
I think we should test non-default values, such as "windows."

Reply:
Modified the tests to use non-default values.